app/javascript/mastodon/actions/notifications.js

```diff
@@ -188,6 +188,7 @@ export function expandNotificationsFail(error, isLoadingMore) {
     type: NOTIFICATIONS_EXPAND_FAIL,
     error,
     skipLoading: !isLoadingMore,
+    skipAlert: !isLoadingMore,
```

Comment:
When we're polling because streaming is down, we first poll home, then notifications, then announcements. If the server is down or network is unavailable, this would mean 3 error alerts. We can skip this one. Home and notifications are always either polling or streaming, so unless we are paginating (`isLoadingMore`) we would not be refreshing notifications outside of that.